tiledb/sm/fragment/fragment_info.h

```diff
@@ -54,26 +54,14 @@ class FragmentInfo {
   /*     CONSTRUCTORS & DESTRUCTORS    */
   /* ********************************* */
 
-  /** Constructor. */
-  FragmentInfo();
```

Comment:
The default constructor should be explicitly deleted, not merely removed.

Reply:
Doh! Good call. Fixed.